### PR TITLE
[AMS-539][AMS-543] migration orders in specified organization unit and fix positions migration

### PR DESCRIPTION
**Pull request by xakep139**
Also fix migration of positions

### CODE REVIEW

File: src/MigrationTool/ImportService.cs [hunk 1]
```diff
@@ -418,6 +451,16 @@ where positions.Contains(pc.MasterPositionId)
             long[] advertisementsToImport;
             using (var context = GetNewContext())
             {
+                OrganizationUnit organizationUnit = null;
+                if (!string.IsNullOrEmpty(_destOrganizationUnitName))
+                {
+                    organizationUnit = context.OrganizationUnits.FirstOrDefault(ou => ou.Name == _destOrganizationUnitName);
```
Comment (denisivan0v) [inline src/MigrationTool/ImportService.cs]: This needs to be approved by testers team

File: src/MigrationTool/ImportService.cs [hunk 2]
```diff
@@ -257,23 +287,26 @@ private void EnsurePositionsAreEqual(PositionDescriptor apiPosition, Position er
...
 
                 files = await (from o in context.Orders
-                                   join op in context.OrderPositions on o.Id equals op.OrderId
-                                   join opa in context.OrderPositionAdvertisement on op.Id equals opa.OrderPositionId
-                                   join adv in context.Advertisements on opa.AdvertisementId equals adv.Id
-                                   join ae in context.AdvertisementElements on adv.Id equals ae.AdvertisementId
-                                   join t in context.AdvertisementTemplates on adv.AdvertisementTemplateId equals t.Id
-                                   join aet in context.AdvertisementElementTemplates on ae.AdvertisementElementTemplateId equals aet.Id
-                                   join f in context.Files on ae.FileId equals f.Id
-                                   where !o.IsDeleted && !o.IsTerminated
```
Comment (denisivan0v) [inline src/MigrationTool/ImportService.cs]: Will we migrate advertising to terminated orders?

Comment (xakep139) [inline src/MigrationTool/ImportService.cs]: Order may have been terminated after placing

File: src/MigrationTool/ImportService.cs (hunk 1 above)
Comment (xakep139) [inline src/MigrationTool/ImportService.cs]: Done in https://github.com/2gis/nuclear-vstore/pull/71/commits/840f5b4f0e5a6d7f46cea6349304c6423b7325e6

**Pull request merged**